kube_metrics_server/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - Kube Metrics Server
 
+## 1.2.2 / 2021-04-20 / Agent 7.27.0
+
+* [Added] Support 0.4.0 metrics server renamed metric names. See [#9202](https://github.com/DataDog/integrations-core/pull/9202).
+
```

Comment:
```suggestion
```
This is automatically updated when the integration gets released

Reply:
ah gotcha. removed 👍